### DISCUSSION TITLE
微信自动查看原图不起作用了

### DISCUSSION THREAD

**Discussion by alumnicn** [gkd-kit/gkd#764] [Q&A]
自动发送原图功能是好的，触发记录里也没有，在应用规则里面都开启了，自动发送原图功能是好的，用的是alsouler和adpro的规则订阅，gkd是最新版本，省电策略设为无限制，并且设置了自启动。
快照链接：https://i.gkd.li/i/17626183
导出日志：https://f.gkd.li/17626141
![1000054066](https://github.com/user-attachments/assets/3e19900e-b2da-405c-a63d-14d853102ec1)
![1000054067](https://github.com/user-attachments/assets/eb563db0-1005-4d65-9b23-312730beac2e)
![1000054068](https://github.com/user-attachments/assets/0eedab6b-4f05-4f68-9fb5-95fdd5ecd83f)

**Comment by lisonge** [comment]
这个你用的是哪个订阅就去哪个订阅反馈，至少目前看起来不是 gkd 的问题

并且你的反馈就一个截图，连界面快照，对应日志都没有，问题是无法解决的

**Reply by alumnicn** [reply]
实在抱歉，导出日志我生成了https://f.gkd.li/17626141

**Reply by lisonge** [reply]
你好，这个规则不是我写的，我的意思是你应该去使用的订阅的仓库反馈

另外建议你去反馈的时候，快照也需要给出

**Reply by alumnicn** [reply]
> 你好，这个规则不是我写的，我的意思是你应该去使用的订阅的仓库反馈
> 
> 另外建议你去反馈的时候，快照也需要给出

谢谢，我换几个规则试试。我意思是你说的对，没有快照看不出来问题。

**Comment by AIsouler** [comment]
@alumnicn 你上面给出的快照截图被替换了，如果涉及到隐私你可以到快照记录里保存快照截图后打码再替换掉快照里的截图，另外你可以到这里反馈
https://github.com/AIsouler/GKD_subscription/issues/new/choose
